validate/validate.go

```diff
@@ -384,6 +384,18 @@ func (v *Validator) CheckLinux() (msgs []string) {
 		msgs = append(msgs, "rootfsPropagation must be empty or one of \"private|rprivate|slave|rslave|shared|rshared\"")
 	}
 
+	for _, maskedPath := range v.spec.Linux.MaskedPaths {
+		if !filepath.IsAbs(maskedPath) {
```

Comment:
maskedPath and readonlyPath means to set paths inside Linux-based container to be masked and readonly in container namespace. In my opinion there is nothing to do with Windows host and there is nothing to do with --host-specific